src/content/fr/2019/caching.md

```diff
@@ -0,0 +1,692 @@
+    <img src="/static/images/2019/16_Caching/ch16_fig16_cacheable_responses_set_cookie.jpg" alt="Figure 16. Réponses cachables avec Set-Cookie." aria-labelledby="fig16-caption" aria-describedby="fig16-description" width="600" height="567">
+  </a>
+  <div id="fig16-description" class="visually-hidden">Le graphique à barres montre que 97&nbsp;% des réponses n'utilisent pas Set-Cookie alors que 3&nbsp;% le font. Ces 3&nbsp;% sont zoomés pour obtenir un autre diagramme à barres montrant la répartition entre 15,3&nbsp;% de réponses <code>private</code>, 84,7&nbsp;% de réponses <code>public</code> pour les ordinateurs de bureau et réciproquement pour les téléphones portables, 18,4&nbsp;% de réponses <code>public</code> et 81,6&nbsp;% de réponses <code>private</code>.</div>
+  <figcaption id="fig16-caption">Figure 16. Réponses pouvant être mises en cache avec <code>Set-Cookie</code>.</figcaption>
+</figure>
+
+## AppCache et service workers
+
+L'Application Cache ou AppCache est une fonctionnalité de HTML5 qui permet aux développeurs et développeuses de spécifier les ressources que le navigateur doit mettre en cache et mettre à disposition des utilisateurs hors ligne. Cette fonctionnalité a été [dépréciée et supprimée des standards du web](https://html.spec.whatwg.org/multipage/offline.html#offline), et sa prise en charge par les navigateurs a diminué. En fait, lorsque son utilisation est détectée, [Firefox v44+ recommande aux développeurs et développeuses d'utiliser plutôt des service workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers). [Chrome 70 limite le cache d'application au seul contexte sécurisé](https://www.chromestatus.com/feature/5714236168732672). Le secteur s'est davantage orienté vers la mise en œuvre de ce type de fonctionnalité avec des service workers - et [la prise en charge des navigateurs](https://caniuse.com/#feat=serviceworkers) a connu une croissance rapide dans ce domaine.
+
+En fait, l'un des [rapports de tendance des archives HTTP montre l'adoption des travailleurs des services](https://httparchive.org/reports/progressive-web-apps#swControlledPages) présenté ci-dessous&nbsp;:
+
+<figure>
+  <a href="/static/images/2019/16_Caching/ch16_fig14_service_worker_adoption.jpg">
+    <img src="/static/images/2019/16_Caching/ch16_fig14_service_worker_adoption.jpg" alt="Figure 17. Série chronologique de pages contrôlées par des service worker." aria-labelledby="fig17-caption" aria-describedby="fig17-description" width="600" height="311">
+    </a>
+  <div id="fig17-description" class="visually-hidden">
+  Un graphique de séries chronologiques montre l'utilisation des sites contrôlés par les service worker d'octobre 2016 à juillet 2019. L'utilisation a augmenté régulièrement au fil des ans, tant pour les téléphones portables que pour les ordinateurs de bureau, mais reste inférieure à 0,6&nbsp;% pour les deux.</div>
```

Comment:
Can you merge this with the line above? Just to keep the row numbers consistent with English version?

Reply:
Ok :) 
It's done